kolena/dataset/evaluation.py

```diff
@@ -181,5 +188,10 @@ def upload_results(
                 df_results = _process_result(config, df_result, id_fields)
                 upload_data_frame(df=df_results, batch_size=BatchSize.UPLOAD_RECORDS.value, load_uuid=load_uuid)
 
-    _upload_results(model, load_uuid, existing_dataset.id)
-    log.info(f"Uploaded results for model '{model}' on dataset '{dataset}'")
+    response = _upload_results(model, load_uuid, existing_dataset.id)
+    log.info(
+        f"uploaded test results for model '{model}' on dataset '{dataset}':\n"
```

Comment:
Can this be single line and more succinct like `n inserted, n updated`? Single line is nicer imo.